src/Meter.java

```diff
@@ -1,4 +1,123 @@
+import java.util.Date;
+import java.util.ArrayList;
 
 public class Meter {
-
+	
+	/**
+	 * The name of this meter.
+	 * Example: "5.67" or "C.127"
+	 */
+	private String name;
+	
+	/**
+	 * Maximum amount of time on this meter in minutes.
+	 */
+	private int timeLimit;
+	
+	/**
+	 * An array list of the times this meter is available for parking.
+	 */
+	private ArrayList<ArrayList<Date>> timeAvailable;
+	
+	/**
+	 * Time left on this meter in minutes.
+	 */
+	private int timeRemaining;
+	
+	/**
+	 * Rate in cents per hour for this meter.
+	 */
+	private int ratePerHour;
+	
+	/**
+	 * Amount of time added for a quarter.
+	 */
+	private int timePerQuarter;
+	
+	/**
+	 * Amount of time added for a dime.
+	 */
+	private int timePerDime;
+	
+	/**
+	 * Amount of time added for a nickel.
+	 */
+	private int timePerNickel;
+	
+	/**Default constructor.
+	 * Creates a new Meter object.
+	 * A meter has a name, time limit, rate per hour, and time available.
+	 * A meter has a time added for quarters dimes and nickels
+	 * 
+	 * @param name 
+	 * @param timeAvailable
+	 * @param timeLimit
+	 * @param timePerQuarter
+	 * @param timePerDime
+	 * @param timePerNickel
+	 */
+	public Meter(
+			String name, 
+			ArrayList<ArrayList<Date>> timeAvailable,
+			int timeLimit, 
+			int timePerQuarter,
+			int timePerDime,
+			int timePerNickel)
+	{
+		this.name = name;
+		this.timeLimit = timeLimit;
+		this.timeAvailable = timeAvailable;
+		this.timePerQuarter = timePerQuarter;
+		this.timePerDime = timePerDime;
+		this.timePerNickel = timePerNickel;
+	}
+	
+	/**
+	 * Determines whether or not this meter has expired.
+	 * @return
+	 * 			True if there is no time remaining,
+	 * 			false otherwise.
+	 */			
+	public boolean isExpired(){
```

Comment:
Can you add a space between `)` and `{`, so, `isExpired() {`. I like that format better.